zookeeper/defaults.yaml

```diff
@@ -0,0 +1,34 @@
+# -*- coding: utf-8 -*-
+# vim: ft=yaml
+---
+zookeeper:
+  pkg:
+    name: zookeeper
+    downloadurl: >
+      https://mirror.csclub.uwaterloo.ca/apache/zookeeper/zookeeper-3.6.2/apache-zookeeper-3.6.2-bin.tar.gz
+    version: 3.6.2
+    installdir: /opt
+
+    # OpenJDK package name on system; defaults to an empty string, which avoids
+    # installing OpenJDK.  Overridden in the `os*map.yaml` files.
+    javajdk: ''
+
+  systemdconfig:
+    user: root
+    group: root
+    limitnofile: 50000
+    limitnproc: 10000
+
+  rootgroup: root
+  config: 'zoo.cfg'
+  service:
+    name: zookeeper
+
+  zookeeperproperties:
+    tickTime: 2000
+    dataDir: ./state/zookeeper
+    clientPort: 2181
+    initLimit: 5
+    syncLimit: 2
+
+    customservers: ''
```

Comment:
Add newline here. https://stackoverflow.com/questions/5813311/no-newline-at-end-of-file